### PR TITLE
Only copy python files when they have changed

### PR DESCRIPTION
**Pull request by joakim-hove**

### CODE REVIEW

**Comment by joakim-hove** [comment]
jenkins build this please

**Comment by joakim-hove** [comment]
jenkins build this please

**Comment by akva2** [comment]
did not do the trick.

File: python/install.py [hunk 1]
```diff
@@ -28,7 +29,8 @@
 
         src_file = os.path.join(path, f)
         target_file = os.path.join(target_path, f)
-        shutil.copy(src_file, target_file)
+        if not os.path.isfile(target_file) or filecmp.cmp(src_file, target_file):
```
Comment (akva2) [inline python/install.py]: need 'not' on the cmp as well.

Comment (joakim-hove) [reply python/install.py]: Yes - I saw; have rewritten slighly

File: python/install.py [hunk 2]
```diff
@@ -28,7 +29,11 @@
 
         src_file = os.path.join(path, f)
         target_file = os.path.join(target_path, f)
-        shutil.copy(src_file, target_file)
+        if not os.path.isfile(target_file)
```
Comment (akva2) [inline python/install.py]: missing :

**Comment by akva2** [comment]
jenkins build this please

**Comment by joakim-hove** [comment]
jenkins build this please

**Comment by akva2** [comment]
now i'm confused. still did not work.

**Comment by akva2** [comment]
jenkins build this please

**Comment by joakim-hove** [comment]
jenkins build this please

**Comment by akva2** [comment]
jenkins build this please

**Comment by joakim-hove** [comment]
Looks like jenkins is rebuilding this:
```
Building CXX object CMakeFiles/opmcommon.dir/src/opm/parser/eclipse/Python/PythonInterp.cpp.o
```

**Comment by akva2** [comment]
yes, the script actually works, there's just another layer of problem.

**Comment by joakim-hove** [comment]
Have made some changes to this in #1173 - that might be the culprit?

**Comment by joakim-hove** [comment]
> Have made some changes to this in #1173 - that might be the culprit?

The last commit - where the `PythonInterp.cpp` is is special cased a bit?

**Comment by akva2** [comment]
i have identified the issue, PR coming soonish.

it's a separate issue. you can remove the print and merge this.

**Comment by joakim-hove** [comment]
> it's a separate issue. you can remove the print and merge this.

Thanks!

**Pull request merged**